src/validator.js

```diff
@@ -57,55 +59,99 @@ function errorPosition(example, action, resource, resourceGroup) {
   return 'in ' + output.join(', ');
 }
 
-module.exports = function (fileName, validateRequests, validateResponses) {
-  fs.readFile(fileName, 'utf8', function (error, data) {
+function lint(file, data, options) {
+  function shouldSkip(event) {
+    return (!options.requireBlueprintName && event.message.indexOf('expected API name') !== -1 );
```

Comment:
is 'expected API name' something special that must be in any apiblueprint? We don't use this in our api blueprint documentation. Could you please point to a reference requiring it?

Where from you get an 'event' here (when it is emitted)?